WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsConnectView.swift

```diff
@@ -0,0 +1,197 @@
+import UIKit
+
+final class CardReaderSettingsConnectView: NSObject {
```

Comment:
Do we really need to extend NSObject? 

The name suffix (`view`) kind of suggest to me this would be a view, and it seems more like a controller/viewmodel?

Reply:
[Copied from slack] NSObject is an interesting one. I only included it because I wasn't able to implement table data source without it. And as you note, its not quite a view and not quite a viewmodel